frontend/src/components/ConfirmationDialog.scss

```
@@ -0,0 +1,56 @@
+.confirmation-dialog-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
```

You can define top/bottom/left/right all as 0 to get this same effect, but it's a little safer in case the parent element does something weird with padding

```suggestion
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
```